src/config.rs

```diff
@@ -146,6 +147,11 @@ impl<'a> Config<'a> {
           .help("Use <WORKING-DIRECTORY> as working directory. --justfile must also be set")
           .requires("JUSTFILE"),
       )
+      .arg(
+        Arg::with_name(arg::INIT)
+          .long("init")
+          .help("Create a default justfile in the project root")
+      )
       .group(ArgGroup::with_name("EARLY-EXIT").args(&[
```

Comment:
Sorry for missing this earlier, `arg::INIT` should be added to this group, which contains subcommand-like arguments that can't be used with one another.